apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx

```diff
@@ -29,4 +30,29 @@ describe("StopPageRedesign", () => {
     render(<StopPageRedesign stopId="123" />);
     expect(screen.queryByText("Test Stop")).not.toBeNull();
   });
+
+  it("should render alerts", () => {
+    const lowAlert: Alert = {
+      updated_at: "Updated: 4/11/2019 09:33A",
+      severity: 7,
+      priority: "low",
+      lifecycle: "upcoming",
+      active_period: [],
+      informed_entity: {} as InformedEntitySet,
+      id: "00005",
+      header: "There is construction at this station.",
+      effect: "other",
+      description: "",
+      url: "https://www.mbta.com"
+    };
+
+    jest
+      .spyOn(useAlertsForStop, "default")
+      .mockImplementation(() => [lowAlert]);
+
+    render(<StopPageRedesign stopId="123" />);
+    expect(
+      screen.queryByText("There is construction at this station.")
+    ).not.toBeNull();
+  });
```

Comment:
suggestion (non-blocking): If you have an interest in additional test for behavior you are already handling, you could add one where `useAlertsForStop` returns `null` and the page still displays properly.

Reply:
The endpoint should be returning an empty list if there are no alerts.  Undefined or null should be an error state, or we are waiting for data to load